packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -16,15 +16,14 @@
 					:label="option.label"
 					@click.stop="handleItemClick(option)"
 				>
-					<!-- eslint-disable-next-line -->
-					<template v-if="areOptionsSelectable" slot="option" :option="option">
```

Comment:
because it's unclear in this context, `option` was never exposed on the scope of the option slot on `IconTextItem` so this was dead all along

It was noticed because the linter (for vue) noticed the shadowing of variables because there's another variable with the same name (option)